sherlock-platform/customization/src/com/google/sherlock/onboarding/EmptyProjectGenerator.java

```diff
@@ -0,0 +1,121 @@
+/***
```

Comment:
Can we (should we?) use Kotlin when we add new classes into the platform?

Reply:
I believe if we are maintaining/implementing/extending an existing Java class, we should stick with Java for consistency and otherwise use Kotlin. Here, we implement `DirectoryProjectGenerator` hence I chose to continue with Java.